package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mdn-translated-content",
```

Comment:
```suggestion
  "name": "@mdn/translated-content",
```

Reply:
This one is marked as private, but doesn't hurt to have it in better shape if that ever changes